Navbars/Fullscreen_Overlay_Navigation/code/css/style.css

```diff
@@ -0,0 +1,411 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:ital@0;1&display=swap');
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: 'Playfair Display', serif;
+  background: #fdfdea;
+  transition: all 2s ease;
+}
+h1 {
+  font-size: 4rem;
+  font-weight: 400;
+}
+h3 {
+  font-size: 1.5em;
+}
```

Comment:
don't add styling directly to HTML tags, use classes instead

Reply:
@thepranaygupta  did the changes 👍.